BlackNectar/UserPreferences.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  UserDefaults.swift
+//  BlackNectar
+//
+//  Created by Kevin Bradbury on 1/18/17.
+//  Copyright © 2017 Black Whole. All rights reserved.
+//
+
+import Foundation
+
+
+class UserPreferences: NSObject {
+    
```

Comment:
I like that it is in its own class and that it is a singleton. 

A few notes:
1. It does not need to be an `NSObject`.
2. Rather than having functions like `setUserPreferences`, it would be more appropriate in this case to use computed properties: https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/Properties.html
To understand what I mean here, see: https://github.com/RedRoma/Lexis/blob/develop/Code/Lexis/Settings.swift